src/internal/converters/ics/ics.go

```diff
@@ -192,108 +199,148 @@ func getRecurrencePattern(
 }
 
 func FromJSON(ctx context.Context, body []byte) (string, error) {
-	data, err := api.BytesToEventable(body)
+	event, err := api.BytesToEventable(body)
 	if err != nil {
 		return "", clues.Wrap(err, "converting to eventable")
 	}
 
 	cal := ics.NewCalendar()
 	cal.SetProductId("-//Alcion//Corso") // Does this have to be customizable?
 
-	id := data.GetId() // XXX: iCalUId?
-	event := cal.AddEvent(ptr.Val(id))
+	id := ptr.Val(event.GetId())
+	iCalEvent := cal.AddEvent(id)
 
-	created := data.GetCreatedDateTime()
+	err = updateEventProperties(ctx, event, iCalEvent)
+	if err != nil {
+		return "", clues.Wrap(err, "updating event properties")
+	}
+
+	exceptionOcurrances := event.GetAdditionalData()["exceptionOccurrences"]
+	if exceptionOcurrances != nil {
+		for _, occ := range exceptionOcurrances.([]any) {
```

Comment:
less nesting this way:
```suggestion
	if exceptionOcurrances == nil {
	  return cal.Serialize(), nil
	}
	
	for _, occ := range exceptionOcurrances.([]any) {
```